arangod/Aql/SingleRemoteModificationExecutor.cpp

```diff
@@ -107,13 +107,14 @@ template <typename Modifier>
 template <typename Modifier>
 auto SingleRemoteModificationExecutor<Modifier>::doSingleRemoteModificationOperation(
     InputAqlItemRow& input, Stats& stats) -> OperationResult {
-  OperationResult result;
   OperationOptions& options = _info._options;
```

Comment:
is the `options` variable needed here at all? it seems that the following code refers to `_info._options` instead.